rust/template/src/main.rs

```diff
@@ -267,6 +270,9 @@ fn main() -> Result<(), String> {
         opt differential_profiler_socket:Option<String>, desc:"Socket address to send Differential Dataflow profiling events. Default (if '--profile-differential' is specified is '127.0.0.1:51318'. Implies '--profile-differential'.";
         opt differential_trace_dir:Option<String>, desc:"Path to a directory to store Differential Dataflow profiling events, e.g., './differential_trace'. Implies '--profile-differential'.";
         opt ddshow:bool=false, desc:"Start 'ddshow' profiler on sockets specified by '--timely-profiler-socket' and (optionally) '--differential-profiler-socket' options. Implies '--timely-profiler'.";
+        opt input:Option<String>, short:'i', desc:"Feed the .json file specified in the argument into the Evaluator.";              // --input
```

Comment:
@mbudiu-vmw , if we cannot feature-gate it, we should at least change the description to clarify that this flag is used for d3log only.

Reply:
In my commenting work I have refrained from making any code changes, assuming that @convolvatron would.
The description I should be able to change, though.